lib/src/runner/node/platform.dart

```diff
@@ -194,9 +194,7 @@ class NodePlatform extends PlatformPlugin
       int socketPort,
       String precompiledPath) async {
     StackTraceMapper mapper;
-    var jsPath =
-        p.join(precompiledPath, p.basename(testPath) + ".node_test.dart.js");
-
+    var jsPath = p.join(precompiledPath, testPath + ".node_test.dart.js");
```

Comment:
single quotes for consistency with the line 2 below.

Would also prefer `'$testPath.node_test.dart.js'`

Reply:
done, in general theres a crazy mix of single/double quotes here so eventually we might want to fix that repo-wide, but at least its consistent within this method now...